src/main/java/homework2/Rectangle.java

```diff
@@ -0,0 +1,10 @@
+package homework2;
+
+public class Rectangle {
+    private int width;
+    private int height;
+
+    static int getArea(int width, int height){
```

Comment:
For this task you need to create a constructor that accepts width and height of the Rectangle and calculates its area (taking parameters from the object not as a method's arguments). It should not be a static method